aws/base-settings/config.tf

```diff
@@ -0,0 +1,39 @@
+#=============================#
+# AWS Provider Settings       #
+#=============================#
+provider "aws" {
+  alias                   = "main_region"
+  region                  = var.region
+  profile                 = var.profile
+  shared_credentials_file = "~/.aws/${var.project}/config"
+}
+
+provider "aws" {
+  alias                   = "secondary_region"
+  region                  = var.region_secondary
+  profile                 = var.profile
+  shared_credentials_file = "~/.aws/${var.project}/config"
+}
+
+provider "vault" {
+  address = var.vault_address
+
+  /*
+  Vault token that will be used by Terraform to authenticate.
+  admin token from https://portal.cloud.hashicorp.com/.
+  */
```

Comment:
@diego-ojeda-binbash OJ maybe here we could replace this for some clearer inline comment.